requirements.txt

```diff
@@ -0,0 +1,2 @@
+nose==1.3.7
```

Comment:
You will have to update `setup.py` also for adding `requirements.txt` to install_requires.